.travis.yml

```diff
@@ -42,3 +51,10 @@ deploy:
     on:
       branch: develop
       node: '8'
+  - provider: script
+    skip_cleanup: true
+    script: bash ./deploy/docker-app-push.sh
+    on:
+      tags: true
+      condition: "$DOCKER_APP_PUSH = 1"
+
```

Comment:
trailing newline

Reply:
Have you seen `.editorconfig`? AFAIK It's good practice, to easily insert something in the future at the EOF :) so it's should stay

Reply:
Unless it's double new line..